### ISSUE TITLE
examples snippets are broken on the biojs registry

### ISSUE THREAD

**Issue by wilzbach** [emepyc/tnt.tree#1]
Hi Miguel,

currently the examples of `tnt.tree` don't work with the BioJS registry

http://biojs.io/d/tnt.tree

I guess that is just because the package on npm is out of sync with github?

**Comment by emepyc** [comment]
Yes, I have updated master without publishing a new npm version.
I don't want to have a new version with every push. Is there a canonical
way of dealing with this in biojs? developing on a different branch?

**Comment by wilzbach** [comment]
> Is there a canonical way of dealing with this in biojs?

No there isn't - go ahead and fix it ;-)

> developing on a different branch?

Sounds reasonable.  An alternative would be to actually use the files packaged in the npm package, but our current form of the registry doesn't store anything.

**Comment by emepyc** [comment]
Thanks Seb,
I'll try to come up with something.
